### DISCUSSION TITLE
Name is close to different project?

### DISCUSSION THREAD

**Discussion by rgetz** [fhdm-dev/scoppy#8] [General]
This looks very cool - I assume it’s coincidence, but are you familiar with: https://github.com/analogdevicesinc/scopy ? 
It been around since ~2017. 

the name is pretty similar to your project (which is completely fine, im not trying to claim trademark/copyright issues), but being in the same space might cause some end user confusion…

if you want - we are open to collaborating - one of our goals is to make scopy (one p) more hardware agnostic than it is today, to support lots of different front ends.

**Comment by fhdm-dev** [comment]
Hi,
Thanks for getting in touch. I wasn't aware of the existance of Scopy so its definitely a coincidence!
I'll put a link to Scopy on the Scoppy github page in case anyone gets there accidentally - tho' Scopy is so much more capable than Scoppy that I don't think the confusion will last long :)

For now at least I think that Scoppy will only support the Pico as the (digital part of the) Front End and of course the Analog section is up to the user. Hopefully a future version will work with external ADCs so that it can run at a higher sample rate - maybe something like the AD7848.

BTW: If you want to contact me directly for any reason you can use the email address that is on the Scoppy app Play Store listing.
